pkg/controller/statusmanager/pod_status.go

```diff
@@ -419,3 +423,14 @@ func (status *StatusManager) listAllStatusObjects() (dss []*appsv1.DaemonSet, de
 	}
 	return
 }
+
+func (status *StatusManager) isClusterRunningOVNKubernetes() bool {
+	daemonSets, _, _ := status.listAllStatusObjects()
+	for _, ds := range daemonSets {
+		dsName := NewClusteredName(ds)
```

Comment:
NIT: Is there a point creating the `dsName`? why don't you get the name straight from `ds`